test/Microsoft.AspNetCore.SignalR.Tests/HubConnectionHandlerTests.cs

```diff
@@ -540,6 +540,35 @@ public void FailsToLoadInvalidTypedHubClient()
             }
         }
 
+        [Fact]
+        public async Task DetailedExceptionEvenWhenNotExplicitlySet()
+        {
+            var serviceProvider = HubConnectionHandlerTestUtils.CreateServiceProvider(builder =>
+            {
+                builder.AddSignalR();
+            });
+
+            var methodName = nameof(MethodHub.ThrowHubException);
+
+            var connectionHandler = serviceProvider.GetService<HubConnectionHandler<MethodHub>>();
+
+            using (var client = new TestClient())
+            {
+                var connectionHandlerTask = await client.ConnectAsync(connectionHandler);
+
+                var message = await client.InvokeAsync(methodName).OrTimeout();
+
+                {
```

Comment:
?

Reply:
These used to be an if/else block here lol